### ISSUE TITLE
Fix the `chrono` RUSTSEC advisory

### ISSUE THREAD

**Issue by hug-dev** [maxlambrecht/rust-spiffe#17]
Hello!

I was receiving a lot of mail from our `cargo audit` nightly checks and finally decided to investigate :). It appears that [the error](https://github.com/parallaxsecond/parsec-client-rust/runs/4638389840?check_suite_focus=true) is coming from the `chrono` crate and the [RUSTSEC-2020-0159](https://rustsec.org/advisories/RUSTSEC-2020-0159) advisory.

I have seen a few PR where people replace `chrono` with `time` (including some of the dependencies). See [here](https://github.com/acw/simple_asn1/pull/26) for example. I suggest that the same is done here too!

**Comment by ionut-arm** [comment]
This was in my todo list for some time, need to finish some other stuff first and then I'll try to get a fix for this.

**Comment by maxlambrecht** [comment]
I got some time to take a stab at this issue. 

I replaced `chrono` dependency by the `time` crate and updated deps `simple_asn1`, `x509_parser` and `zeroize` to versions that no longer depend on chrono. See #21 

What is missing is updating `jsonwebtoken` to 0.8, this version removed the chrono crate. I'm waiting to do that until this [PR](https://github.com/nhynes/jwk-rs/pull/7) on the `jsonwebkey` crate is merged.
